easybuild/easyconfigs/l/libgeotiff/libgeotiff-1.5.1-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'libgeotiff'
+version = '1.5.1'
+
+homepage = 'https://directory.fsf.org/wiki/Libgeotiff'
+description = """Library for reading and writing coordinate system information from/to GeoTIFF files"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/OSGeo/libgeotiff/archive']
+sources = [{
+    'download_filename': '%(version)s.tar.gz',
+    'filename': SOURCE_TAR_GZ
```

Comment:
@fizwit Rather than download `'%(version)s.tar.gz'` and rename it, we should be downloading the prepared source tarball `libgeotiff-%(version)s.tar.gz` instead (like we were doing before), which also alleviates the need for running `autoreconf`?
See also https://github.com/OSGeo/libgeotiff/releases

Reply:
fixed